test/integration_debug_test.go

```diff
@@ -516,3 +525,7 @@ func TestDependencyGraphWithMultiInclude(t *testing.T) {
 	assert.Contains(t, stdoutStr, `"main" -> "depb";`)
 	assert.Contains(t, stdoutStr, `"main" -> "depc";`)
 }
+
+func isWindows() bool {
```

Comment:
NIT: I think we should put `isWindows` in helpers if we're going to reuse it outside this file.